src/api.rs

```diff
@@ -74,6 +63,8 @@ impl RawSpan {
         env: &Option<String>,
         cfg: &ApmConfig,
     ) -> RawSpan {
+        let http_enabled = span.tags.contains_key(&"http.url".to_string());
```

Comment:
I am not sure, but do you need the `to_string()` here and below?

Reply:
I think you're right.  Will change.